src/test/java/com/williamfiset/algorithms/datastructures/dynamicarray/DynamicArrayTest.java

```diff
@@ -50,6 +50,48 @@ public void testIndexOutOfBounds4() {
     list.removeAt(-66);
   }
 
+  @Test(expected = Exception.class)
```

Comment:
Probably best to check that the exception is IndexOutOfBoundsException rather than the generic Exception

Reply:
Fixed as requested :)